data/border_points.js

```diff
@@ -1,47 +1,48 @@
 var border_points = {
-    "type": "FeatureCollection",
```

Comment:
@lynnnakiganda , what change was made in this file?

Reply:
added a new border point called kerwa